application/frontend/config/main.php

```diff
@@ -4,6 +4,13 @@
 
 /* Get frontend-specific config settings from ENV vars or set defaults. */
 $frontCookieSecure = Env::get('FRONT_COOKIE_SECURE', true);
+// default to match docker-compose.yml settings
```

Comment:
What do you think about using the getArrayFromPrefix approach for arrays of configs like this instead of explicitly getting each parameter as separate vars? I don't know if this is cleaner, but it could allow providing more configuration to each server if needed/supported and would work for one or two memcache servers in configuration. 

```php
$memcache1 = Env::getArrayFromPrefix('MEMCACHE_CONFIG1_');
$memcache2 = Env::getArrayFromPrefix('MEMCACHE_CONFIG2_');
$memcacheConfig = [];
if ( is_array($memcache1) && ! empty($memcache1) {
  $memcacheConfig[] = $memcache1;
}
if ( is_array($memcache2) && ! empty($memcache2) {
  $memcacheConfig[] = $memcache2;
}
```

Then in sessionCache/servers config you can just have:

```php
'servers' => $memcacheConfig
```
I think that'll make it easier to support one or two memcache servers.